### DISCUSSION TITLE
meson.py build problem

### DISCUSSION THREAD

**Discussion by danishraza0912** [su2code/SU2#2134] [General]
![image](https://github.com/su2code/SU2/assets/110195851/f73ff284-3cce-434c-9a41-931fb9c8469c)
I am trying to run the command ./meson.py build but I am getting this error . Also the process does not ends even after so much time.  Can anyone help ??

**Comment by kursatyurt** [comment]
It looks like you are trying to install a remote disk (cloud) or trying to run from a remote disk location. You can use your local disk. Unfortunately, I cannot test in Windows and have not touched it for many years.
